src/Stratis.Bitcoin.Features.RPC/RPCFeature.cs

```diff
@@ -89,6 +90,7 @@ public static IFullNodeBuilder AddRPC(this IFullNodeBuilder fullNodeBuilder, Act
             {
                 features
                 .AddFeature<RPCFeature>()
+                .DependOn<MempoolFeature>()
```

Comment:
Hmm its odd ther eis a dependency on mempool here

Reply:
Yeah, fixed it. 
RPC feature project had mempool project in dependency projects list so I added mempool. But since mempool depend on consensus there was no bug. 

I've removed mempool project dependency and added consensus + changed depend on to depend on consensus in the latest commit.